### ISSUE TITLE
Need process to generate 32bit snaps for the Pi

### ISSUE THREAD

**Issue by oparoz** [kyrofa/owncloud-snap#30]
In my exprience, the Pi device is not the best thing to use to build images since compiling MySQL crashes often when only 1GB of RAM is available, so how do we generate test images from PRs that we can test?

@kyrofa @jospoortvliet

**Comment by kyrofa** [comment]
Great question, this will be a fun one!

First of all, let's be explicit: when you say image, what do you mean? An image that can be flashed on the rpi, or a built snap? Both are possible, but the latter is quite a bit easier than the former.

**Comment by oparoz** [comment]
Good catch :). I meant snap, but I wrote image. What we need right now is a way to test new version of the ownCloud snap. I see the Pi image as a delivery truck which we shouldn't have too much to worry about, except maybe for things like swap, hw config, credentials, etc.

**Comment by kyrofa** [comment]
Yeah, same page. And this is totally doable with the Launchpad builders using the Launchpad Python API via Travis.

**Comment by oparoz** [comment]
ETA?

**Comment by kyrofa** [comment]
> ETA?

Perhaps the better question right now is: what's the priority relative to the other issues? High so we can test fixes for those issues?

**Comment by oparoz** [comment]
I'd say high, yes, because otherwise we'll either end up shipping untested snaps or not shipping anything at all, but having a much improved snap for amd64...
cc @jospoortvliet

**Comment by kyrofa** [comment]
Bringing our IRC conversation to a close here: For now we're manually using the Launchpad builders, though ideally this would be done with a GitHub integration (e.g. Travis).
